test/examples/cypress/e2e/hello_world.cy.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+describe('Hello World example app', () => {
+  it('says hello to user', () => {
+    cy.visitKibana('/app/helloWorld');
+
+    cy.getByTestSubj('helloWorldDiv').as('resultContainer');
+    cy.getByTestSubj('helloWorldTextHandle').as('textHandle');
+    cy.getByTestSubj('helloWorldResetHandle').as('resetHandle');
+
+    // initial
+    cy.get('@resultContainer').contains('Hello Worlff!');
```

Comment:
I can't find where we're changing "World!" to "Worlff!" and assume this is a typo:
```suggestion
    cy.get('@resultContainer').contains('Hello World!');
```

Reply:
The commit that changed "World" to "Worlff" confuses me. What was the purpose other than making sure ci does report a failure?

Reply:
@TinaHeiligers yes, I wanted to ensure that a CI failure fails "in the right way," and I noticed an issue: the Artifacts tab is empty. I received a hint to look at `https://github.com/elastic/kibana/blob/main/.buildkite/scripts/lifecycle/post_command.sh#L12` to connect job failure artifacts with downloads from the build workspace